static/app/components/core/text/text.mdx

```diff
@@ -204,7 +204,7 @@ Use `monospace` for fixed-width text.
 
 <Storybook.Demo>
   <Storybook.SideBySide vertical>
-    <Flex align="center" gap="xl" style={{marginBottom: '16px'}}>
+    <Flex align="center" gap="xl" marginBottom="2xl">
```

Comment:
shouldn’t we wrap this in a `Stack` instead of using margin ourselves? leading by example :)